CHANGELOG.md

```diff
@@ -0,0 +1,39 @@
+Change log
+===
+
+### Newer Versions 
+
+For newer versions of the plugin, see [GitHub releases](https://github.com/jenkinsci/matlab-plugin/releases)

```

Comment:
A period is missing. Please terminate the sentence with a period.

Reply:
Done.